docs/download.rst

```diff
@@ -65,6 +65,46 @@ which used to manage reporting symbols that can't be found during
 processing. See https://bugzilla.mozilla.org/show_bug.cgi?id=1361809
 
 
+Microsoft on-the-fly Symbol Lookups
+===================================
+
+Under certain conditions, if a symbol can not be found in S3, we might
+try to look it up from Microsoft's download server
+(``https://msdl.microsoft.com/download/symbols/``) if the symbol file
+ends in ``.pdb`` and filename ends in ``.sym``.
```

Comment:
Is this correct?
It filters out a bunch of files that are called things like `foo.pdb/HEXHEX/foo.dll` which I see a lot of in the S3 prod logs.